### PR TITLE
Code Highlight fixes

### PR DESCRIPTION
**Pull request by ChrisChinchilla**

### CODE REVIEW

File: src/crate/theme/rtd/conf/crash.py [hunk 1]
```diff
@@ -22,6 +22,8 @@
 from crate.theme.rtd.conf import *
 
 project = u'Crate Shell'
+primary_domain = 'Crate.Crash'
```
Comment (chaudum) [inline src/crate/theme/rtd/conf/crash.py]: does the domain have any impact on the output?

Comment (ChrisChinchilla) [inline src/crate/theme/rtd/conf/crash.py]: @chaudum Not yet, but it will do in the future.

**Comment by nslater** [comment]
@chaudum @ChrisChinchilla what's the status of this? do we wanna merge it in? perhaps split it into two separate changes, if one is uncertain?

**Comment by chaudum** [comment]
css changes may still be relevant. the custom domain should not be changed.

**Pull request closed without merging**